sentences/HDG.js

```diff
@@ -9,13 +9,24 @@ const nmea = require('../nmea.js')
 module.exports = function (app) {
   return {
     title: 'HDG - Heading magnetic:.',
-    keys: ['navigation.headingMagnetic'],
-    f: function hdg (headingMagnetic) {
+    keys: ['navigation.headingMagnetic', 'navigation.magneticVariation' ],
+    defaults: [undefined, ''],
+    f: function hdg (headingMagnetic, magneticVariation) {
+      var magneticVariationDir = '';
+      if ( magneticVariation != '' ) {
+        magneticVariationDir = 'E';
+        if ( headingMagnetic < 0 ) {
+          magneticVariationDir = 'W';
+          magneticVariation =   Math.abs(magneticVariation);
```

Comment:
Spacing.

Reply:
I prefer always to write code so that function arguments are never reassigned. I find code easier to understand that way. https://github.com/airbnb/javascript#functions--reassign-params.